src/haggadah/events.cljs

```diff
@@ -133,13 +132,14 @@
  ::add-haggadah
  (fn [{:keys [db]} [_ title]]
    {::add-haggadah! {:path ["users" (:uid db) "haggadot"]
-                     :haggadah (assoc dsl/haggadah :title title :createdAt (js/Date.))
+                     :haggadah (assoc dsl/default-haggadah :title title :createdAt (js/Date.))
                      :on-success (re-frame/dispatch [::push-state :haggadah-success])
                      :on-error (keyword->func ::error)}}))
 
 (re-frame/reg-fx
  ::add-haggadah!
  (fn [{:keys [path haggadah on-success on-error]}]
+   (println "Here is the path to add a haggadah" path)
```

Comment:
Do you still need the `println` ?

Reply:
I did not realize that was still there. I will remove it.